Repo URL for `academic` theme updated

gcushen opened the pull request:
Hello guys

My `academic` repository has changed location from `github.com/gcushen/hugo-academic` to github.com/wowchemy/starter-academic 

Please can you kindly review and merge the PR - I have updated the corresponding submodule url in `.gitmodules` to reflect the change.

Keep up the awesome work 🙌

Thanks 🙏

George

digitalcraftsman commented:
Hello George,

I've had a look at the new repository at https://github.com/wowchemy/starter-academic. From its README alone is hard to deduce how this repository is related to Hugo or that it's Hugo theme. The repo description 

> Easily create a beautiful website using Wowchemy, GitHub, and Netlify 

and the headline of the README

> Academic Template for Wowchemy Website Builder

make it more sound like a template specific to your website builder service Wowchemy. This makes it hard for users to see if this theme is only usable in combination with Wowchemy or could still be used as a stand-alone theme for Hugo. 

The old remote url now points to https://github.com/wowchemy/wowchemy-hugo-modules which seems to be the original Academic theme repo that you've submitted a while ago. It's not entirely clear for me (and probably visitors / users) how the two repositories are related.

In addition, the README of https://github.com/wowchemy/starter-academic leaves more the impression of being a huge advertisement for Wowchemy. I have nothing against commercial services that target Hugo and Hugo themes as market. However, it should be transparent for users what they're facing and how it can be used (only in combination with Wowchemy?).

Hence I suggest to improve the wording in https://github.com/wowchemy/starter-academic to address the aforementioned points and to advertise Wowchemy a bit more discreetly.

gcushen commented:
Thank you for your feedback @digitalcraftsman :)

Based on your feedback, I have updated the [Hugo Academic Readme's](https://github.com/wowchemy/starter-academic) headline and description to clearly mention Hugo a number of times and link to it a couple of times.

I have also updated the Repo Description and URL to clearly mention Hugo and point directly to the demo of the Hugo template.

Bep inspired me to convert the theme from the standard Git submodule system to the new Hugo Module system, which allows for monorepos of modules and a lot of other opportunities. It will be interesting to see if Hugo Modules takes off (it seems that the most popular themes have stuck with the traditional Git approach) and if the themes site expands to further promote Hugo's Module system.

As for commercialisation, the two repos you mentioned are both MIT open source repos with GitHub Sponsors enabled on them. The Wowchemy site itself currently provides documentation for getting started with using Hugo and the template, along with similar GitHub Sponsors link. Hence, the main change has been restructuring the repos in GitHub and converting to a new modular approach using Hugo Modules.

digitalcraftsman commented:
Thanks for explaining the difference between the two repositories. The README now reads much more neutrally.
